core/utxo/topsort_test.go

```diff
@@ -180,3 +181,20 @@ func TestTopSortWithoutCircleCase4(t *testing.T) {
 	t.Log("circle->", circle)
 	t.Log("childDAG->", childDAG)
 }
+
+func TestSplitBlock(t *testing.T) {
+	tx1 := &pb.Transaction{Txid: []byte("tx1")}
+	tx2 := &pb.Transaction{Txid: []byte("tx2"), TxInputs: []*pb.TxInput{&pb.TxInput{RefTxid: []byte("tx1")}}}
```

Comment:
What if tx2's input refers to tx777 ?
```
tx2 := &pb.Transaction{Txid: []byte("tx2"), TxInputs: []*pb.TxInput{&pb.TxInput{RefTxid: []byte("tx777")}}}
```
Or is this situation won't be occurred as the miner do topologic sort for txs when mine block?

Reply:
won't be occurred